downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -81,6 +81,37 @@ To view bug fixes, see the GitHub milestone for Swan Lake Update 9 (2201.9.0) of
 
 ### Improvements
 
+#### `cloud` package
+- Directories can now be mounted as Configmaps and Secrets 
+
 ### Bug fixes
 
 ## Backward-incompatible changes
+
+### `cloud` package
+
+#### Cloud.toml related changes
+
+- SSL configurations are not automatically retrieved from the code anymore. You need to explicitly mark them as secrets in Cloud.toml. 
```

Comment:
We usually just have bullet points here, but then again it's usually just lang.

How about we add `### `cloud` package` (and similarly for language changes, we will add a heading) and then just have bullet points without another level of headers?

Reply:
I've suggested this in https://github.com/ballerina-platform/ballerina-dev-website/pull/8876/files#diff-7ee6bb79765f14c3dff835d98a4c33179eedea8962b3fc2da71bb5670d0ff67dR125